firmware/build/extract-mrenclave

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
+function print_usage() {
+    echo "Usage: $0 <enclave_binary> <config_file>"
+    echo ""
+    echo "Options:"
+    echo "  enclave_binary:    path of an enclave binary file."
+    echo "  config_file:       configuration file specifying the enclave properties."
+    echo "                     refer to the oesign sign --help for the list of properties."
+    echo ""
+    echo "Description:"
+    echo "  This script extracts the MRENCLAVE and the DIGEST values from the enclave"
+    echo "  binary and prints them to stdout. The script can be used both for unsigned"
+    echo "  and signed enclave binaries."
+    echo ""
+    echo "  Signed binaries:"
+    echo "    The MRENCLAVE and DIGEST are calculated from the signed enclave binary and"
+    echo "    the enclave properties specified in the configuration file. Both values are"
+    echo "    printed in hexadecimal format to stdout."
+    echo ""
+    echo "  Unsigned binaries:"
+    echo "    The DIGEST is calculated from the unsigned enclave binary and the enclave"
+    echo "    properties specified in the configuration file. The MRENCLAVE is set to zero."
+    echo "    Both values are printed in hexadecimal format to stdout."
+}
+
+if [[ $# -lt 2 ]]; then
```

Comment:
We need to allow for a single argument (signed enclave binary case).

Reply:
Just pushed a commit with that change, let me know how it looks